apps/dashboard/pages/participant-database/index.tsx

```diff
@@ -243,13 +247,56 @@ const Index = () => {
 
   return (
     <Wrapper>
-      <StyledButton
-        onClick={() => {
-          router.replace('/sponsor-booth');
-        }}
-      >
-        <Image width="30" height="30" src={'/arrow.svg'} alt="Illustration" />
-      </StyledButton>
+      <MenuBar>
+        <StyledButton
+          onClick={() => {
+            router.replace('/sponsor-booth');
+          }}
+        >
+          <Image width="30" height="30" src={'/arrow.svg'} alt="Illustration" />
+        </StyledButton>
+        <DownloadButton
+          onClick={async () => {
```

Comment:
nit: Extract this onclick into a separate function so that the component tree is easy to read